File: LC1295.cpp
class Solution {
public:
    int getcount(int n){
        int c = 0;
        while(n> 0){
            n/= 10;
            c++;
        }
        return c;
    }

    int findNumbers(vector<int>& nums) {
        int c = 0;
        for(auto i : nums)
            if(!(getcount(i)%2))
                c++;
        
        return c;
    }
};

File: LC1295.py
class Solution:
    def getcount(self, n):
        c = 0

        while n > 0:
            n //= 10
            c += 1
        return c

    def findNumbers(self, nums: List[int]) -> int:
        return sum([self.getcount(i)%2 == 0 for i in nums])


# Another way

class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        return sum([len(str(i))%2 == 0 for i in nums])
        
